packages/core/src/actionTypes.ts

```diff
@@ -19,3 +19,4 @@ export const error = ActionTypes.ErrorCustom;
 export const update = ActionTypes.Update;
 export const choose = ActionTypes.Choose;
 export const pure = ActionTypes.Pure;
+export const actionGroup = ActionTypes.ActionGroup;
```

Comment:
I wouldn't introduce this one. I feel like arrays are already pretty descriptive when it comes to such groups and we can use them directly.

Reply:
I added this following @davidkpiano's comment about action groups being syntactic sugar for `xstate.actionGroup` over here: https://github.com/statelyai/xstate/discussions/3554#discussioncomment-3499861

If we didn't use an `ActionObject` to describe these, would we just flatten the group arrays in `toActionObjects`?